src/main/java/org/opentripplanner/hasura_client/hasura_objects/Area.java

```diff
@@ -0,0 +1,26 @@
+package org.opentripplanner.hasura_client.hasura_objects;
+
+import java.util.List;
+
```

Comment:
To jest  reprezentacja obiektu hasurowego, tak? Jeśli nie to dodaj komentarz/zmień nazwę na bardziej mówiącą

Reply:
Doczytałem do zmian w ParkingZone i zrozumiałem. Komentarz jednak by się przydał